radio/src/edgetx.cpp

```diff
@@ -401,6 +402,10 @@ void generalDefault()
 
 #if defined(MANUFACTURER_RADIOMASTER)
   g_eeGeneral.audioMuteEnable = 1;
+  g_eeGeneral.stickMode = 1; // Default to mode 2
```

Comment:
This disables the function of `DEFAULT_MODE`. I'd suggest removing it.

Reply:
The discussion is currently whether or not we just switch the default mode to `2`, as we think this is what most users have.